pkg/reduce/data_forward.go

```diff
@@ -366,8 +366,23 @@ func (df *DataForward) forwardAChunk(ctx context.Context) {
 			m.Message.Keys = []string{dfv1.DefaultKeyForNonKeyedData}
 		}
 		m.Watermark = time.Time(processorWM)
+	}
+
+	var dataMessages = make([]*isb.ReadMessage, 0, len(readMessages))
+
+	// store the offsets of the messages we read from ISB
+	var readOffsets = make([]isb.Offset, len(readMessages))
+	for idx, m := range readMessages {
```

Comment:
do we really have to do this?

Reply:
you are right readOffsets is not used anywhere in the method